cli/bgc-backend/cmd/worker.go

```diff
@@ -67,7 +67,7 @@ func newPrometheusScope(c prometheus.Configuration) tally.Scope {
 	scopeOpts := tally.ScopeOptions{
 		CachedReporter: reporter,
 		Separator:      prometheus.DefaultSeparator,
-		Prefix:         "background_checks",
+		Prefix:         "",
```

Comment:
This line can go as empty `Prefix` is the default.

Reply:
Makes sense to remove the prefix, but lets make sure there is a label so we can tell these metrics are from the background check app.